doc-content/enterprise-only/weblogic/wls-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,286 @@
+[id='wls-configure-embedded-engine-proc']
+ifdef::PAM[]
+= Configuring an embedded {PROCESS_ENGINE} and {DECISION_ENGINE} in {WEBLOGIC}.
+endif::[]
+
+ifdef::DM[]
+= Configuring an embedded {DECISION_ENGINE} in {WEBLOGIC}
+endif::[]
+
+An embedded engine is a light-weight work-flow engines and rule engine which executes your decisions and business processes. They can be embedded as part of your application or deployed as a service possibly in the cloud.
+
+ifdef::PAM[]
+This section describes how to setup an embedded {PROCESS_ENGINE} and {DECISION_ENGINE} in {WEBLOGIC}.
```

Comment:
This procedure describes how to configure...
Would you set up both at the same time? Otherwise, use and instead of or.